backend/tex/aux_parser.py

```diff
@@ -0,0 +1,44 @@
+import argparse
```

Comment:
pls reformat this file according to [PEP 8](https://www.python.org/dev/peps/pep-0008/). you can use [this] (http://pep8online.com/) to check if your code meets the pep8 requirements